### ISSUE TITLE
[Safari] Dark reader stays on after being turned off

### ISSUE THREAD

**Issue by piotrtobolski** [darkreader/darkreader#2335]
I think it is similar to #2065 but worse. It happens for me on random webpages but most often Google and other complex web apps. You can see that is also happens on webpages with custom domain and port. Switching back and forth (extension or blacklist) or refreshing the page helps sometimes but often don't. The only thing that always works is disabling extension in Safari preferences.

I use Dark Reader with setting "disable for all web pages by default" and selectively turn it on. I had it turned on for those web pages for a moment but changed it back after noticing issues.

Also, can we somehow get access to the raw blacklist file in the Safari version? I think I would be able to fix broken entries by myself.

<img width="590" alt="googlemaps-auto" src="https://user-images.githubusercontent.com/1545279/79732748-636de900-82f4-11ea-8b69-07ffe3ae27a3.png">
<img width="1347" alt="synology" src="https://user-images.githubusercontent.com/1545279/79732752-649f1600-82f4-11ea-9902-72a1697c286e.png">
<img width="638" alt="googlemaps-off" src="https://user-images.githubusercontent.com/1545279/79732736-5f41cb80-82f4-11ea-9dc9-a24b89e75d9b.png">

**Comment by Gusted** [comment]
Last time I checked Safari development is closed-source

**Comment by piotrtobolski** [comment]
It is true but I have noticed Safari/Catalina issues being reported here too

**Comment by Gusted** [comment]
Yeah no problems with reporting the issue here. It can take a long time as Darkreader v5 is in the planning.

**Comment by xerus2000** [comment]
I have the same problem in Firefox. 
For example on https://overpass-turbo.eu/ - I opened the page with DarkReader on, turned it off but the site is always darkened when I switch to another tab and back to it. 
The only way to fix this is to reload the page (seems to fix it permanently) or turn DarkReader off and on again (which only fixes it until I switch to another tab and back).

**Comment by xerus2000** [comment]
Actually, this seems to happen on every page, as long as I don't reload it DarkReader darkens it again if I switch to it.

**Comment by Gusted** [comment]
Could you make a GIF of it? As I'm having a hard time understanding

**Comment by xerus2000** [comment]
I'm not yet sure how to reproduce it, but it does happen from time to time, I will get back when I know more.

**Comment by xerus2000** [comment]
Okay, I had it happen again to me: https://mega.nz/file/dZ5n3BBD#gQ9tlfEMFB6n4a5ii-O1hUZ1BiXe_eCkVMad13X_bQQ
Unfortunately it didn't record the DarkReader popup there, but I simply switched it off.
After accidentally reloading the page with Ctrl+Shift+R it was fixed, here I'm doing the same as previously, but this time it works: https://mega.nz/file/4Q5xwRCY#7RV_61RUgq0OiWG31ShcPEV6rY8DEgtjE9268KnvlwE

**Comment by sn0wyfall** [comment]
@xerus2000 Maybe you need to @ him to notify him

**Comment by Gusted** [comment]
#2927 no need to mention

**Comment by phoenixeliot** [comment]
I'm getting this same issue with the Chrome version; should that get a new issue?

**Comment by Gusted** [comment]
Yes. Safari and Chrome are different codebases.

**Comment by sn0wyfall** [comment]
Is there a place to report Safari-related bugs?

**Comment by Gusted** [comment]
> Is there a place to report Safari-related bugs?

Just open up a issue here.

**Comment by joshinils** [comment]
possible duplicate of #5723 ?
